curriculum/challenges/english/02-javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-projects/cash-register.md

```diff
@@ -180,6 +180,29 @@ assert.deepEqual(
 );
 ```
 
+Global variables should not be used.
+
+```js
+
+checkCashRegister(19.5, 20, [["PENNY", 0.5], ["NICKEL", 0], ["DIME", 0], ["QUARTER", 0], ["ONE", 0], ["FIVE", 0], ["TEN", 0], ["TWENTY", 0], ["ONE HUNDRED", 0]])
+
+assert.deepEqual(
+  checkCashRegister(19.5, 20, [
+    ['PENNY', 0.01],
+    ['NICKEL', 0],
+    ['DIME', 0],
+    ['QUARTER', 0],
+    ['ONE', 1],
+    ['FIVE', 0],
+    ['TEN', 0],
+    ['TWENTY', 0],
+    ['ONE HUNDRED', 0]
+  ]),
+  { status: 'INSUFFICIENT_FUNDS', change: [] }
+);
+```
+
```

Comment:
Minor quibble usually we have one line here I think, not two. 

Otherwise I like this fix.